includes/Checker/Checks/Plugin_Repo/Plugin_Header_Fields_Check.php

```diff
@@ -155,6 +170,40 @@ public function run( Check_Result $result ) {
 			}
 		}
 
+		if ( empty( $plugin_header['Version'] ) ) {
+			$this->add_result_error_for_file(
+				$result,
+				sprintf(
+					/* translators: %s: plugin header field */
+					__( 'The "%s" header is missing in the plugin file.', 'plugin-check' ),
+					esc_html( $labels['Version'] )
+				),
+				'plugin_header_missing_plugin_version',
+				$plugin_main_file,
+				0,
+				0,
+				__( 'https://developer.wordpress.org/plugins/plugin-basics/header-requirements/', 'plugin-check' ),
+				7
+			);
+		} else {
+			if ( preg_match( '|[^\d\.]|', $plugin_header['Version'] ) ) {
+				$this->add_result_error_for_file(
+					$result,
+					sprintf(
+						/* translators: %s: plugin header field */
+						__( 'The "%s" header in the plugin file should only contain numeric and period characters.', 'plugin-check' ),
```

Comment:
Why? 🤔 That sounds way to strict. Many plugins use versions such as 14.0.0-RC1 or so, and semver is not that restrictive.

Reply:
Yah, I have also thought about that. Even Akismet has beta is some versions. Did I misread the code? I took reference from here https://github.com/WordPress/wordpress.org/blob/trunk/wordpress.org/public_html/wp-content/plugins/plugin-directory/shortcodes/class-upload-handler.php#L294

Reply:
Ah, it makes sense to block this in uploader as initial submission is not expected to have alphabets in the version.

Reply:
Lets make regex less strict by this: `/^[0-9a-z.-]+$/i`